openapi-diff/src/core/OpenApiDiff.Core/Logging/ObjectPath.cs

```diff
@@ -33,6 +34,19 @@ private ObjectPath(IEnumerable<Func<JToken, string>> path)
             return list == null || list.Count == 0 ? null : list[0].i.ToString();
         });
 
+        public static string PathName(string path)
```

Comment:
i know this is just an extracted method, but is there anything pointing to this regex ? ( i.e. is it documented anywhere )